BUILD

```diff
@@ -1,23 +0,0 @@
-java_binary(
-    name = "json_test",
-    main_class= "com.vaticle.typedb.example.genealogy.math_genealogy.json_test",
-    srcs=["json-test.java"],
-    deps = ["@maven//:org_sharegov_mjson","@vaticle_typedb_client_java//:client-java",
-                                                                "@vaticle_typedb_client_java//api",
-                                                              "@vaticle_typeql//java:typeql-lang",
-                                                              "@vaticle_typeql//java/query"],
-    data =
-        [
-            ":data.json",
-            ":datalite.json"
-        ]
-)
-java_test(
-    name="DataLoaderTest",
```

Comment:
same here

Reply:
But Java does not allow hyphens in variable names, so if we want to call this class from other files, we wont be able to?